src/hotspot/share/opto/library_call.cpp

```diff
@@ -5363,6 +5366,102 @@ void LibraryCallKit::create_new_uncommon_trap(CallStaticJavaNode* uncommon_trap_
   uncommon_trap_call->set_req(0, top()); // not used anymore, kill it
 }
 
+//------------------------------inline_array_partition-----------------------
+bool LibraryCallKit::inline_array_partition() {
+
+  address stubAddr = nullptr;
+  const char *stubName;
+  stubName = "array_partition_stub";
```

Comment:
It could one line.

Reply:
Fixed the assignment. Please see the latest commit that was pushed.